Source/Settings.as

```diff
@@ -49,6 +49,18 @@ vec2 Setting_General_WheelsPos = vec2(0.909f, 0.9f);
 
 [Setting category="General" name="Wheels size"]
 vec2 Setting_General_WheelsSize = vec2(230, 370);
+
+[Setting category="General" name="Show acceleration"]
+bool Setting_General_ShowAcceleration = true;
```

Comment:
I think by default make this false, just like the wheels element. It might be weird if people upgrade and they suddenly have a new widget that they're not sure of what it does.